spec/factories/users.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :user do
+    first_name "MyString"
```

Comment:
Cool. I think its a good idea to go with whatever you're most comfortable with for now. We can standardize our test setups with factory girl when we refactor.